vcluster/deploy/topologies/isolated-workloads.mdx

```diff
@@ -28,48 +46,76 @@ policies:
+For all options and more information take a look at the
+[policies](/docs/vcluster/configure/vcluster-yaml/policies/) documentation.
 
 <Policies/>
 
 ## Advanced Isolation
 
-Besides this basic workload isolation, you could also dive into more advanced isolation methods, such as isolating the workloads on separate nodes or through another container runtime.
+Besides this basic workload isolation, you could also dive into more advanced
+isolation methods, such as isolating the workloads on separate nodes or through
+another container runtime.
 
-You should also be aware that pods created in the vCluster will set their [tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/), which will affect scheduling decisions. To prevent the pods from being scheduled to the undesirable nodes you can use the `sync.fromHost.nodes.selector.labels` option or admission controller as mentioned above.
+You should also be aware that pods created in the vCluster will set their
+[tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/),
+which will affect scheduling decisions. To prevent the pods from being scheduled
+to the undesirable nodes you can use the `sync.fromHost.nodes.selector.labels`
+option or admission controller as mentioned above.
 
 ### Workload & Network Isolation within the vCluster
 
-The above mentioned methods also work for isolating workloads inside the vCluster itself, as you can just deploy resource quotas, limit ranges, admission controllers and network policies in there. To allow network policies to function correctly, you'll need to enable this in vCluster itself though.
+The above mentioned methods also work for isolating workloads inside the vCluster
+itself, as you can just deploy resource quotas, limit ranges, admission controllers
+and network policies in there. To allow network policies to function correctly,
+you'll need to enable this in vCluster itself though.
 
 ### Secret based Service Account tokens
 
-By default vCluster will create Service Account Tokens for each pod and inject them as an annotation in the respective pods
-metadata. This can be a security risk in certain scenarios. To mitigate this there's an option in vCluster
-which creates separate secrets for each pods Service Account Token and mounts it accordingly using projected volumes. This option
-is not enabled by default but can be enabled on demand. To enable you can use:
+By default vCluster will create Service Account Tokens for each pod and inject
+them as an annotation in the respective pods metadata. This can be a security
+risk in certain scenarios. To mitigate this there's an option in vCluster
+which creates separate secrets for each pods Service Account Token and mounts it
+accordingly using projected volumes. This option is not enabled by default but
+can be enabled on demand. To enable you can use:
```

Comment:
Replace `To enable you can use:` to `To enable this option, enable the configuration in the vcluster.yaml:`